src/main/java/org/opentripplanner/model/Timetable.java

```diff
@@ -58,28 +55,30 @@ public class Timetable implements Serializable {
 
   private final TripPattern pattern;
 
-  private final List<TripTimes> tripTimes = new ArrayList<>();
+  private final List<TripTimes> tripTimes;
 
-  private final List<FrequencyEntry> frequencyEntries = new ArrayList<>();
+  private final List<FrequencyEntry> frequencyEntries;
```

Comment:
Investigate if we can have both tripTimes and frequencyEntries!

Reply:
This is possible and this seems to be handle accordingly in the code.
I do not think we need to enforce that a timetable should contain only one or the other.